Pyopenms CMake fails if you have __pycache__ folders from a previous test execution

Probably only affects developers but we should fix our _copy_assets function nonetheless.

```
CMake Error at src/pyOpenMS/CMakeLists.txt:59 (configure_file):
  configure_file input location

    /git/OpenMS-fixes-src/src/pyOpenMS/tests/integration_tests/__pycache__

  is a directory but a file was expected.
Call Stack (most recent call first):
  src/pyOpenMS/CMakeLists.txt:365 (_copy_assets)
```

--- a/src/pyOpenMS/CMakeLists.txt
+++ b/src/pyOpenMS/CMakeLists.txt
@@ -56,7 +56,10 @@ function(_copy_assets _asset_dir _regex _target_dir)
   foreach(_asset_file ${_assets})
     # make path relative
     file(RELATIVE_PATH _relative_path ${_asset_dir} ${_asset_file})
-    configure_file(${_asset_file} ${_target_dir}/${_relative_path} COPYONLY)
+    # check if the asset is a directory
+    if(NOT IS_DIRECTORY ${_asset_file})
+      configure_file(${_asset_file} ${_target_dir}/${_relative_path} COPYONLY)
+    endif()
   endforeach()
 endfunction()
 

